resources/mod_php.rb

```diff
@@ -0,0 +1,39 @@
+include Apache2::Cookbook::Helpers
+unified_mode true
+
+property :module_name, String,
+         default: lazy { apache_mod_php_modulename },
+         description: 'Module name for the Apache PHP module.'
+
+property :so_filename, String,
+         default: lazy { apache_mod_php_filename },
+         description: 'Filename for the module executable.'
+
+action :create do
+  # install mod_php package
+  package apache_mod_php_package unless apache_mod_php_package.nil?
```

Comment:
We really should have two additional parameters for this resource to account for the different ways an admin can install mod_php (especially on CentOS).
1. We should have a ``package_name`` parameter which defaults to ``apache_mod_php_package``. However if a user is using something such as IUS repositories, the name will be slightly different and we should be able to use different names
2. I think we should also have a ``install_package`` true/false (defaults to ``true``) to allow folks who install this via source or using some other means external to this cookbook.